apps/kfp-tekton/upstream/third-party/application/cluster-scoped/application-crd.yaml

```diff
@@ -146,7 +146,7 @@ spec:
                     type: array
                   maintainers:
                     description: Maintainers is an optional list of maintainers of
-                      the application. The maintainers in this list maintain the the
+                      the application. The maintainers in this list maintain the
```

Comment:
This i not the right repository. Please change it in the upstream repository where this file is synchronized from.

Reply:
Thanks! Modified

Please review again.